devtools/data/gen-serialized-results.py

```diff
@@ -71,18 +82,20 @@ def generate_md_json(smc):
 def generate_ahfe_json(smc):
     settings = AbsoluteSolvationProtocol.default_settings()
     settings.solvent_simulation_settings.equilibration_length = 10 * unit.picosecond
-    settings.solvent_simulation_settings.production_length = 500 * unit.picosecond
+    settings.solvent_simulation_settings.production_length = 10 * unit.picosecond
     settings.vacuum_simulation_settings.equilibration_length = 10 * unit.picosecond
-    settings.vacuum_simulation_settings.production_length = 1000 * unit.picosecond
+    settings.vacuum_simulation_settings.production_length = 10 * unit.picosecond
```

Comment:
I've tried this several times below 1 ns and it fails on the regular. It ends up wasting a lot more time re-running everything than just running the full nanosecond.